sdk/include/psdk/uxtheme.h

```diff
@@ -15,7 +15,7 @@ extern "C" {
 #define DTBG_COMPUTINGREGION        0x00000010
 #define DTBG_MIRRORDC               0x00000020
 #define DTT_GRAYED                  0x00000001
-#if DLL_EXPORT_VERSION >= _WIN32_WINNT_VISTA
+#if defined(_UXTHEME_PCH_) || (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
```

Comment:
This UXTHEME_PCH has nothing to do in public headers as well...
(I think i understand why you used it there though...)
But if you add vista+ functions in our uxtheme to be able to use them actively there, why not compiling uxtheme with its _WIN32_WINNT defined to _WIN32_WINNT_VISTA (or a higher value)?
As follows:
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/shell32/CMakeLists.txt;hb=90de2b660b12919c2f60f2f31ad41e37c123d662#l9
```
remove_definitions(-D_WIN32_WINNT=0x502)
add_definitions(-D_WIN32_WINNT=0x600)
```
(Note the `remove_definitions` before in order to remove the old value.)